src/UI/templates/default/Glyph/tpl.glyph.html

```diff
@@ -0,0 +1,80 @@
+<a class="glyph<!-- BEGIN highlighted --> highlighted<!-- END highlighted -->" <!-- BEGIN with_action -->href="{ACTION}"<!-- END with_action --> aria-label="{LABEL}">
+
+
+<!-- BEGIN settings -->
+<span class="glyphicon glyphicon-cog" aria-hidden="true"></span>
+<!-- END settings -->
+
+<!-- BEGIN collapse -->
+<span class="glyphicon glyphicon-triangle-bottom" aria-hidden="true">Issue: New Bootstrap Version (3.3.6) Needed</span>
+<!-- END collapse -->
+
+<!-- BEGIN expand -->
+<span class="glyphicon glyphicon-triangle-right" aria-hidden="true">Issue: New Bootstrap Version (3.3.6) Needed</span>
+<!-- END expand -->
+
+<!-- BEGIN add -->
+<span class="glyphicon glyphicon-plus-sign" aria-hidden="true"></span>
+<!-- END add -->
+
+<!-- BEGIN remove -->
+<span class="glyphicon glyphicon-minus-sign" aria-hidden="true"></span>
+<!-- END remove -->
+
+<!-- BEGIN up -->
+<span class="glyphicon glyphicon-circle-arrow-up" aria-hidden="true"></span>
+<!-- END up -->
+
+<!-- BEGIN down -->
+<span class="glyphicon glyphicon-circle-arrow-down" aria-hidden="true"></span>
+<!-- END down -->
+
+<!-- BEGIN back -->
+<span class="glyphicon glyphicon-chevron-left" aria-hidden="true"></span>
+<!-- END back -->
+
+<!-- BEGIN next -->
+<span class="glyphicon glyphicon-chevron-right" aria-hidden="true"></span>
+<!-- END next -->
+
+<!-- BEGIN sortAscending -->
+<span class="glyphicon glyphicon-arrow-up" aria-hidden="true"></span>
+<!-- END sortAscending -->
```

Comment:
Why all these blocks? I would prefer one generic template with generic blocks for the glyphs. If you need to change general glyph HTML in the future you do not want to do that for every single glyph.